popde/density_estimate.py

```diff
@@ -120,6 +120,68 @@ def prepare_data(self):
         else:
             self.kde_data = self.std_data
 
+
+    def evaluate_with_transf(self, points):
+        """
+        Transforms the input points in the same way as the KDE training data,
+        evaluates the KDE on the transformed points and adjusts the KDE
+        values by the Jacobian of the transformations.
+
+        Parameters:
+        -----------
+        points : array-like, shape (n_samples, n_features)
+            The original parameter space points to be transformed and evaluated.
+
+        Returns:
+        --------
+        kde_vals : array-like, shape (n_samples,)
+        The KDE values adjusted by the Jacobian of the transformations.
+        """
+        ### Transform points with transf, normalize, rescale
+        if self.input_transf is not None:
+            transf_points = transf.transform_data(points, self.input_transf)
+        else:
+            transf_points = points
+        #normalize
+        if self.stdize:
+            std_transf = ['stdize' for dim in self.ndim]
+            std_values_for_points = np.std(transf_points, axis=0)  # record the stds
+            std_points = transf.transform_data(transf_points, std_transf)
+        else:
+            std_values_for_points = None
+            std_points = transf_points
+
+        #rescale
+        if self.rescale is not None:
+            transf_data = transf.transform_data(std_points, self.rescale)
+        else:
+            transf_data = std_points
+
+        #Evaluate kde on transform points
+        kde_vals = self.evaluate(transf_data)
+
+        num_points, num_dims = transf_data.shape
+        if len(self.input_transf) != num_dims:
+            raise ValueError("The transformation list length must match the number of dimensions in eval_vals")
+      
+        #get Jacobians using points in original units
+        for i, option in enumerate(self.input_transf):
+            print(i, type(option))
+            if option in['log', 'ln']:
+                print("option is ", option)
+                # Apply log transformation for that variable
+                kde_vals *= 1.0 / points[:, i]
+            elif option == 'exp':
+                # Apply exponential transformation
+                kde_vals *= np.exp(points[:, i])
+            elif option =='none':
```

Comment:
you should allow for both 'none' and 'None' - see https://github.com/tdent/pop-de/blob/main/popde/transform_utils.py#L35